index.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+	$('#myForm').on('submit', function(e) {
+		e.preventDefault();
+		$("#output").empty();
+		var searchTerm = $("#searchTerm").val();
+		var url = 
+		"https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser=" 
+		+ searchTerm + "&ucdir=older&uclimit=5&callback=?"; 
+		$.ajax({
+			url: url,
+			type: 'GET',
+			contentType: "application/json; charset=utf-8",
+			async: false,
+			dataType: "jsonp",
+			headers: { 'Api-User-Agent': 'useredits-tool/1.0' },
+			success: function(data, status, jqXHR) {
+				if(data.error != null) {
+					alert(data.error.info);
+				}
+				else {
+					arr = data.query.usercontribs;
+					var str = "";
+					for(var i=0;i<arr.length;i++) {
+						str +=
+						"<div class='panel panel-default col-xs-4' > <div class='panel-body'>Title: " 
+						+ arr[i].title + " (<a href='https://en.wikipedia.org/w/index.php?" + 
+						arr[i].title.replace(/ /g,"_") + "&oldid=" + arr[i].revid + "&diff=prev'>diff</a>)"
```

Comment:
It is not safe to use a title in an URL without escaping. What happened if I registered a user called `"onmouseover="alert('xss')`, for example? (Using it in HTML, as in the line above, is probably safe as `<>` characters are forbidden in titles. Still a good practice to escape it.

Reply:
Yes. 
Replaced title from the URL with pageid.

Reply:
Are you sure this works? I don't think we have a pageid parameter, it's called curid. And using the title with proper escaping would be a nicer solution. You'll also need to escape the one that gets displayed to the user (in a different way; URLs and text in a HTML page require different escaping).

Reply:
Actually we only need the revid.